src/Morpho.sol

```diff
@@ -340,6 +340,11 @@ contract Morpho is IMorphoStaticTyping {
 
     /* LIQUIDATION */
 
+    struct Vars {
+        uint256 collateralPrice;
+        uint256 liquidationIncentiveFactor;
+    }
```

Comment:
I hate this change, but I haven't been able to do something else...

Reply:
You can do the same thing as [here](https://github.com/morpho-labs/morpho-blue-private/pull/9)